python/revng/internal/daemon/__init__.py

```diff
@@ -135,6 +135,7 @@ def make_startlette() -> Starlette:
 
     if DEBUG:
         print(f"Manager workdir is: {manager.workdir}")
+        signal.signal(signal.SIGUSR2, lambda s, f: event_manager.save())
```

Comment:
1. Why do we need this?
2. What do we use `SIGUSR1` for?

Reply:
1. For debugging purposes, sometimes I want to test things that happen during save (e.g. S3, `store` function, etc.) and I'd like to avoid shutting down the server or waiting 5 minutes of inactivity
2. To reload opened files, references: [1](https://github.com/encode/uvicorn/blob/3dd0d44445cc2d96c77037ba1f7bfefd288f370e/uvicorn/workers.py#L75) [2](https://github.com/benoitc/gunicorn/blob/88fc4a43152039c28096c8ba3eeadb3fbaa4aff9/gunicorn/workers/base.py#L187)

Reply:
Gotcha, maybe we can merge SIGUSR2 into SIGUSR1, it doesn't sound harmful.

Reply:
Don't get me wrong, we can, but nesting signal handlers isn't the nicest thing and `USR2` is unused

Reply:
Nesting? Just let the SIGUSR1 signal handler do both things. Can we?

Reply:
Not directly, you need to do something like:
```python
old_handler = signal.getsignal(signal.SIGUSR1)  # get the old handler
def new_handler(sig, frame):
  save_stuff()
  old_handler(sig, frame)
signal.signal(signal.SIGUSR1, new_handler)
```